pyodide-build/pyodide_build/buildall.py

```diff
@@ -430,7 +431,15 @@ def builder(n):
     )
 
 
-def generate_packages_json(pkg_map: dict[str, BasePackage]) -> dict:
+def generate_package_hash(full_path: Path) -> str:
```

Comment:
Let's make this private,

```suggestion
def _generate_package_hash(full_path: Path) -> str:
```